confidential-data-hub/storage/test/mountinfo.json

```diff
@@ -0,0 +1,21 @@
+{  "volume-type":"alibaba-cloud-oss",
+    "device":"/var/lib/kubelet/pods/xxxx-xxxx-xxxx-xxxx/volumes/kubernetes.io~csi/pv-oss-direct-assigned/mount",
+    "fstype":"oss",
+    "metadata":
+    {
+        "akId":"sealed.XXX",
+        "akSecret":"sealed.XXX",
+        "annotations":"",
+        "bucket":"oss",
+        "encrypted":"gocryptfs",
+        "encPasswd":"sealed.XXX",
+        "kmsKeyId":"",
+        "otherOpts":"-o max_stat_cache_size=0 -o allow_other",
+        "path":"/",
+        "readonly":"false",
+        "targetPath":"/var/lib/kubelet/pods/xxxx-xxxx-xxxx-xxxx/volumes/kubernetes.io~csi/pv-oss-direct-assigned/mount",
+        "url":"oss-cn-hangzhou.aliyuncs.com",
+        "volumeId":"pv-oss-direct-assigned"
+    },
+    "options":["-o","max_stat_cache_size=0","-o","allow_other"]
+}
```

Comment:
I think it is better to include this test json in the concrete implementation of `fn secure_mount` PR, or it would be confusing to other reviewers